styles.css

```
@@ -0,0 +1,100 @@
+@import url("https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #222;
+
+  font-family: "Rubik", sans-serif;
+
+  min-height: 100vh;
+  /* width: 100%; */
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  color: #fff;
+}
+
+main.container {
+  position: relative;
+  max-width: 100%;
+
+  margin: 1em;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-template-rows: repeat(3, minmax(200px, auto));
+
+  text-align: center;
+
+  gap: 1em;
+
+  grid-auto-flow: dense;
+}
+
+.box {
+  background-color: #333;
+
+  padding: 1em;
+  display: grid;
+
```

Don't leave space between two line of similar properties

```suggestion
    Display:grid;
    Place-items:center;
```